frontend/src/components /summarys.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+// const url = "http://localhost:5000/summary";
+const url = "https://anna-sarah-auth-project.herokuapp.com/summary";
+
+export const Summary = props => {
+  const [message, setMessage] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  //Getting the accessToken from the browser's localStorage
+  //and sending it as the header "Authorization"
+  const accessToken = window.localStorage.getItem("accessToken");
+
+  useEffect(() => {
+    setErrorMessage("");
+    fetch(url, {
+      method: "GET",
+      headers: { Authorization: accessToken }
+    })
+      .then(res => {
+        if (!res.ok) {
+          throw new Error("Access denied", JSON);
+        } else {
+          return res.json();
+        }
+      })
+      .then(json => setMessage(json.message))
+      .catch(err => {
+        setErrorMessage(err.message);
+      });
+  }, [accessToken]);
+
+  return (
+    <>
+      <div>
+        <h1>Welcome!</h1>
+        <h3>You have successfully signed in!</h3>
+        <h5>{message}</h5>
+        <Button
+          id="logout"
+          className="btn"
+          onClick={() => (window.location.href = "/SignIn")}
+          type="button"
+        >
+          logout
+        </Button>
+      </div>
+      {errorMessage && <div>{errorMessage}</div>}
```

Comment:
Maybe make this clearer that it's an error message and hide the successful message?
![image](https://user-images.githubusercontent.com/807352/75664882-845e7a00-5c73-11ea-91b7-1ed2ca17afc5.png)